src/api_utils/gladia_api_utils/image_management.py

```diff
@@ -1,11 +1,29 @@
+import base64
 import os
+from io import BytesIO
 
 import cv2
 import numpy as np
 from PIL import Image
 from skimage.filters import gaussian
 
 
+def Image_to_base64(image: Image) -> str:
```

Comment:
```suggestion
def convert_image_to_base64(image: Image) -> str:
```

Maj are only for CONST and Classes